kie-dmn-feel/pom.xml

```diff
@@ -109,15 +105,6 @@
         </executions>
       </plugin>
 
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.5.1</version>
```

Comment:
@psiroky true that maven-compiler is duplicated, but I believe complete removal would default to Java 8 up to the 
```
  <groupId>org.jboss</groupId>
  <version>20</version>
  <artifactId>jboss-parent</artifactId>
```

sorry but I don't know if that is intended, so I preferred to highlight.

Besides of this comment, for me +1 for the whole PR and thanks!